### PR TITLE
Review7 guides

### PR DESCRIPTION
**Pull request by jstaffor**
.../topics/pro-contribute-guide.adoc                          | 11 -----------
.../topics/pro-keycloak-enablement.adoc                       |  2 --
.../topics/pro-keycloak-implementation.adoc                   | 10 ----------
.../topics/pro-passportauth-pointing-to-a-datasource.adoc     |  9 ---------
.../topics/pro-running-the-demo-repositories.adoc             |  7 -------
.../topics/pro-using-the-demo-app.adoc                        |  6 ------
.../topics/pro-working-with-templates-on-rhmap.adoc

### CODE REVIEW

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 1]
```diff
@@ -68,7 +68,7 @@ NOTE: The terminal is out of use while the Core Server is running.
 
 [id='{context}-published-repositories-running-the-mobile-and-portal-applications']
 [discrete]
-== Running the Mobile and Portal {PROJECT_NAME} Applications
+== Running the Mobile and Portal Starter Applications
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: After consultation with team, community etc. we decided on the names here.
We trying to stick with with `Reference Application` or just `Application` adding another option will be confusing. Starter seems off.

FYI @austincunningham @JameelB @witmicko

File: docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc [hunk 2]
```diff
@@ -77,15 +77,15 @@ Open a new terminal and execute the following command in folder `raincatcher-por
 npm run start
 ----
 
-Both the _Mobile_ and _Portal_ demo applications open automatically in a browser.
+Both the _Mobile_ and _Portal_ Starter Applications open automatically in a browser.
```
Comment (wtrocki) [inline docs/workforce-management-framework/topics/pro-running-the-demo-repositories.adoc]: Same as above

**Review by wtrocki — COMMENTED** [review]
Great change. Just one minor comment to be addressed. Everything looks really good.

**Review by wtrocki — APPROVED** [review]
Reviewed! Rendered docs locally.
Good to merge

**Review by jstaffor — COMMENTED** [review]
@wtrocki - 'Starter' name removed from content

**Pull request merged**